draft-irtf-cfrg-vdaf.md

```diff
@@ -3020,8 +3029,24 @@ removing it after unsharding.
 
 # IANA Considerations
 
-This document makes no request of IANA.
+A codepoint for each (V)DAF in this document is defined in the table below. Note
```

Comment:
You probably want to structure this section a little differently.  You're providing instructions to IANA: Create X, Y, Z, registries; add entries to Q registry; etc.  So I would have a top-level summary here, and then a subsection for each registry you're creating / touching.  You should be able to basically copy/paste the prose you need from MLS:

https://messaginglayersecurity.rocks/mls-protocol/draft-ietf-mls-protocol.html#name-iana-considerations

Reply:
Thanks, @bifurcation! The immediate need for codepoints is to address #102. I filed #110 to address this later on.